var/spack/repos/builtin/packages/scorep/package.py

```diff
@@ -34,17 +34,22 @@ class Scorep(Package):
     homepage = "http://www.vi-hps.org/projects/score-p"
     url      = "http://www.vi-hps.org/upload/packages/scorep/scorep-2.0.2.tar.gz"
 
+    version('3.0',   '44da8beaa3f71436a5f6fe51938aab2f')
     version('2.0.2', '8f00e79e1b5b96e511c5ebecd10b2888')
     version('1.4.2', '3b9a042b13bdd5836452354e6567f71e')
     version('1.3',   '9db6f957b7f51fa01377a9537867a55c')
 
     ##########
     # Dependencies for SCORE-P are quite tight. See the homepage for more
     # information.
+    # SCOREP 3
+    depends_on('otf2@2:', when='@3:')
+    depends_on('opari2@2:', when='@3:')
+    depends_on('cube@4.3:', when='@3:')
     # SCOREP 2.0.2
-    depends_on('otf2@2.0', when='@2.0.2')
-    depends_on('opari2@2.0', when='@2.0.2')
-    depends_on('cube@4.3:4.4', when='@2.0.2')
+    depends_on('otf2@2.0', when='@2.0.2:')
+    depends_on('opari2@2.0', when='@2.0.2:')
+    depends_on('cube@4.3:4.4', when='@2.0.2:')
```

Comment:
These may conflict because `scorep@3.0` meets the criteria for both `@2.02:` and `@3.0:`.